tests/Makefile.am.inc

```diff
@@ -109,7 +109,7 @@ tests/test-%.wrap:
 
 tests/runtime-repo: tests/make-test-runtime.sh flatpak
 	rm -rf tests/runtime-repo
-	PATH=$$(cd $(top_builddir) && pwd):$${PATH} tests/make-test-runtime.sh tests/runtime-repo org.test.Platform ""
+	PATH=$$(cd $(top_srcdir)/tests && pwd):$${PATH} make-test-runtime.sh tests/runtime-repo org.test.Platform ""
```

Comment:
Yeah, that looks right. I don't even know why the path is being adjusted, though. The script can be called directly with `$(top_srcdir)/tests/make-test-runtime.sh`. Why have the shell find the program when you know exactly where it is?

FWIW, the whole `$$(cd $(top_somedir) && pwd)` seems silly, too. The same thing should be available with `$(abs_top_srcdir)`. But I know that's not what you're hunting here.